src/deps-x86_64.txt

```diff
@@ -1,3 +1,6 @@
+# For grub install when creating images without anaconda, for now only on x86_64
+grub2
```

Comment:
Shouldn't this be added to `src/deps-aarch64.txt` as well?

Reply:
If you insist I will add it here, but I guess that it is pulled via the "other" grub packages as mentioned in the comment this is for the anacondaless install, which is AFAIK only on x86_64 for now.

Reply:
no, arm64 is anaconda-less too

Reply:
Done